site/docs/patterns/metric.mdx

```diff
@@ -0,0 +1,156 @@
+---
+title: Metric
+layout: DetailPattern
+aliases:
+  - /salt/patterns/metric
+data:
+  resources:
+    [
+      { href: "https://storybook.saltdesignsystem.com/?path=/story/patterns-metric--metric", label: "Examples" },
+      { href: "https://www.figma.com/file/TI8zR16jnisL4W7ANedtUJ/(WIP)-Metric?type=design&node-id=652-3336&mode=design&t=UpxMBoYUQ8vMxz6z-0", label: "Metric examples", internal: true },
+    ]
+---
+
+Metric is a pattern that displays an important number value prominently. You can add indicators or labels to help provide context to the presented value.
+
+  <Diagram
+    src="/img/patterns/metric/metric.png"
+    alt="Two example metrics displaying positive and negative indicators."
+    contentPosition="top"
+    border
+  />
+
+We provide design guidance for using Salt [`Icon`](/salt/components/icon), [`Link`](/salt/components/link), [`StackLayout`](/salt/components/stack-layout) and [`Text`](/salt/components/text) components to build the pattern. You can also find implementation examples in the [Storybook](https://storybook.saltdesignsystem.com/?path=/story/patterns-metric--metric).
+
+## When to use it
+
+Use the pattern to give more prominence to a key measure. Metrics are often used as building blocks in dashboards. They direct users to key information that they may otherwise overlook, particularly when multiple numerical values are displayed. Metrics can encompass a broad range of quantitative data, such as financial statistics, performance indicators and user engagement.
+
+You can add multiple metrics to your UI, and you can use different sizes to indicate hierarchy when dealing with multiple metrics of varying importance.
```

Comment:
Not sure you need the first bit, probs just need the but about using different sizes

Reply:
This sounds like it should be a best practice and be rephrased to something like:

```suggestion
When dealing with multiple metrics of varying importance you can use different sizes to indicate hierarchy.
```